.github/workflows/test.yml

```
@@ -8,6 +8,17 @@ permissions:
   contents: read
 
 jobs:
+  # This is umbrella job for all tests needed to pass to make it simpler
+  # to maintain GitHub Actions status required checks since job names and
+  # amount change over the time and it is easier to maintian having just
+  # this umbrella check set as required.
+  status_check:
+    name: All required tests passing check
+    needs: [rails]
```

Other examples I see just say,
```suggestion
    needs: rails
```
It seems not to be completing though all the other test runs are complete.

---

This works as intended. We just need to change repo settings to check for this new check once merged.

---

I have merged this and updated the check.